app/assets/stylesheets/application.css

```diff
@@ -102,9 +102,15 @@ a.social-link:hover {
 }
 
 .nav-link:hover {
-	color: inherit;
+	color: white !important;
```

Comment:
White is way clear and I personally like how it looks with the dark nav bar.

Reply:
Are you saying it's good or that I should change it?